<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>milan338</author>
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(43, 47, 59);</string>
  </property>
  <widget class="QPushButton" name="btn_menu_effects">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	background-color: rgb(27, 27, 35);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(74, 74, 74);
}

QPushButton:pressed 
{
	background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>  Effects</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/assets/icons/ico_effects.png</normaloff>:/icons/assets/icons/ico_effects.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_menu_settings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	background-color: rgb(27, 27, 35);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(74, 74, 74);
}

QPushButton:pressed 
{
	background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string>  Settings</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/assets/icons/ico_settings.png</normaloff>:/icons/assets/icons/ico_settings.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="bar_left" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(27, 27, 35);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QWidget" name="bar_bottom" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>540</y>
     <width>831</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 43, 54);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5"/>
  </widget>
  <widget class="QPushButton" name="btn_effect_off">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>550</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(39, 43, 54);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(51, 58, 72);
}

QPushButton:pressed 
{
	background-color: rgb(51, 76, 100);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/assets/icons/ico_power.png</normaloff>:/icons/assets/icons/ico_power.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_device_information">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>550</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(39, 43, 54);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(51, 58, 72);
}

QPushButton:pressed 
{
	background-color: rgb(51, 76, 100);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/assets/icons/ico_information.png</normaloff>:/icons/assets/icons/ico_information.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_list_device">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(27, 27, 35);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(74, 74, 74);
}

QPushButton:pressed 
{
	background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/assets/icons/ico_devices.png</normaloff>:/icons/assets/icons/ico_devices.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_list_strip">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>550</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(27, 27, 35);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(74, 74, 74);
}

QPushButton:pressed 
{
	background-color: rgb(89, 89, 89);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/assets/icons/ico_strips.png</normaloff>:/icons/assets/icons/ico_strips.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_device_debug">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>550</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(39, 43, 54);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(51, 58, 72);
}

QPushButton:pressed 
{
	background-color: rgb(51, 76, 100);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/assets/icons/ico_debug.png</normaloff>:/icons/assets/icons/ico_debug.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QStackedWidget" name="main_menus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>681</width>
     <height>541</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="menu_effects">
    <widget class="QScrollArea" name="effects_scroll_region">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>661</width>
       <height>541</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(43, 47, 59);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="effects_scroll_region_widgets">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>661</width>
        <height>541</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="main_menu_button_layout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_menu_effect_new">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	background-color: rgb(51, 58, 72);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(51, 76, 100);
}

QPushButton:pressed 
{
	background-color: rgb(90, 164, 253);
}</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_menu_effect_reset">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color:rgb(43, 47, 59);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(51, 58, 72);
}

QPushButton:pressed 
{
	background-color: rgb(51, 76, 100);
}</string>
         </property>
         <property name="text">
          <string>Reset Layout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="menu_settings">
    <widget class="QScrollArea" name="settings_scroll_region">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>661</width>
       <height>551</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(43, 47, 59);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="settings_scroll_region_widgets">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>661</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="settings_toggle_advanced" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(39, 43, 54);
border-radius: 10px;</string>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>3</y>
            <width>631</width>
            <height>102</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="settings_toggle_advanced_layout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="tmp_layout_settings_advanced"/>
           </item>
           <item>
            <widget class="QLabel" name="label_settings_advanced">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="text">
              <string>Advanced Mode</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="settings_button_layout"/>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="context_menus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>121</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(27, 27, 35);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="context_device">
    <widget class="QScrollArea" name="context_devices">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(39, 43, 54);
border-radius: 10px;
</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>191</height>
       </rect>
      </property>
      <widget class="QPushButton" name="stub_device">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	background-color: rgb(39, 43, 54);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(51, 76, 100);
}

QPushButton:pressed 
{
	background-color: rgb(90, 164, 253);
}</string>
       </property>
       <property name="text">
        <string>stub device</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="context_strip">
    <widget class="QScrollArea" name="context_strips">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(39, 43, 54);
border-radius: 10px;
</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>191</height>
       </rect>
      </property>
      <widget class="QPushButton" name="stub_strip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	background-color: rgb(39, 43, 54);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(51, 76, 100);
}

QPushButton:pressed 
{
	background-color: rgb(90, 164, 253);
}</string>
       </property>
       <property name="text">
        <string>stub strip</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_version">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>550</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	background-color: rgb(39, 43, 54);
	border-radius: 10px;
}

QPushButton:hover
{
	background-color: rgb(51, 58, 72);
}

QPushButton:pressed 
{
	background-color: rgb(51, 76, 100);
}</string>
   </property>
   <property name="text">
    <string>Version 0.0.1</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QSlider" name="slider_brightness">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>561</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="toolTipDuration">
    <number>2</number>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider
{
	background-color: rgb(39, 43, 54);
}

QSlider::groove:horizontal
{ 
	background-color: rgb(27, 27, 35);
	height: 10px; 
	border-radius: 4px;
}

QSlider::handle:horizontal
{ 
	background-color: rgb(51, 58, 72);
	border: 2px solid rgb(51, 58, 72);
	width: 16px; 
	height: 16px;
	line-height: 20px; 
	margin-top: -5px; 
	margin-bottom: -5px; 
	border-radius: 10px; 
}

QSlider::handle:horizontal:hover
{ 
	background-color: rgb(51, 76, 100);
	border: 2px solid rgb(51, 76, 100);
}

QSlider::handle:horizontal:pressed
{ 
	background-color: rgb(90, 164, 253);
	border: 2px solid rgb(90, 164, 253);
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>main_menus</zorder>
  <zorder>bar_bottom</zorder>
  <zorder>bar_left</zorder>
  <zorder>btn_menu_effects</zorder>
  <zorder>btn_menu_settings</zorder>
  <zorder>btn_list_device</zorder>
  <zorder>btn_list_strip</zorder>
  <zorder>context_menus</zorder>
  <zorder>btn_version</zorder>
  <zorder>btn_device_debug</zorder>
  <zorder>btn_device_information</zorder>
  <zorder>btn_effect_off</zorder>
  <zorder>slider_brightness</zorder>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
